### PR TITLE
Contrôle a posteriori : desambiguation des labels pour les DDETS pendant la phase contradictoire

### PR DESCRIPTION
**Pull request by vincentporte**
### Quoi ?

Rendre plus explicite les labels pour les DDETS pendant la phase contradictoire.
+ suppression du label "en attente" générateur d'ambiguité pour la SIAE.

### Pourquoi ?

Certains labels : 
* laissaient penser que le passage en phase contradictoire avait été annulé pour la SIAE et que par conséquent, tout le travail déjà réalisé par la DDETS était à refaire,
* ne permettaient pas aux DDETS de retrouver rapidement les dernières pièces à valider

### Comment ?

* mise à jour des templates
* inversion des contrôles dans la méthode `state` de `EvaluatedJobApplication`

### CODE REVIEW

File: itou/siae_evaluations/models.py [hunk 1]
```diff
@@ -490,6 +490,20 @@ def state(self):
...
+            eval_admin_crit.submitted_at is None for eval_admin_crit in self.evaluated_administrative_criteria.all()
+        ):
+            return evaluation_enums.EvaluatedJobApplicationsState.UPLOADED
+
+        if any(
+            eval_admin_crit.review_state == evaluation_enums.EvaluatedAdministrativeCriteriaState.PENDING
+            for eval_admin_crit in self.evaluated_administrative_criteria.all()
+        ):
+            return evaluation_enums.EvaluatedJobApplicationsState.SUBMITTED
+
```
Comment (vincentporte) [inline itou/siae_evaluations/models.py:506]: Cette inversion n'impacte pas le test unitaire. Elle fait suite à une précédente inversion pour tenter de résoudre un edge case. Mais qui en a créé un autre. Les cas nominaux des tests restent conformes.

File: itou/templates/siae_evaluations/includes/siae_infos.html [hunk 2]
```diff
@@ -7,15 +7,15 @@ <h3 >
     <div class="col-lg-3 col-md-3 col-4">
         <p class="small mb-0">
             {% if evaluated_siae.state == "SUBMITTED"%}
-                <p class="badge badge-pilotage float-right">À traiter</p>
+                <p class="badge badge-pilotage float-right">{%if evaluated_siae.reviewed_at%}Phase contradictoire - {%endif%}À traiter</p>
```
Comment (rsebille) [inline itou/templates/siae_evaluations/includes/siae_infos.html:10]: Petite typo dans le nom du commit ;)

File: itou/templates/siae_evaluations/includes/criterion_infos.html [hunk 3]
```diff
@@ -5,9 +5,7 @@ <h3>
         </h3>
     </div>
     <div class="col-md-3 mt-1 text-right">
-        {% if review_state == "PENDING" %}
-            <p class="text-success"><i class="ri-checkbox-circle-line"></i>En attente</p>
```
Comment (rsebille) [inline itou/templates/siae_evaluations/includes/criterion_infos.html:9]: Donc pour les critères en attente on n'affiche plus rien, c'est pas un peu radical ?
Les utilisateurs ne vont-il pas se demander dans quel état est le critère ou vont supposer qu'il est accepté/validé ?

**Review by rsebille — APPROVED** [review]

File: itou/templates/siae_evaluations/includes/criterion_infos.html (hunk 3 above)
Comment (vincentporte) [reply itou/templates/siae_evaluations/includes/criterion_infos.html:9]: Oui. C'était la raison de l'ajout. Sera revu en UX.
